tensorflow/lite/examples/android/app/README.md

```diff
@@ -17,9 +17,9 @@ for our external and internal code to merge.
 1. Follow the [Bazel steps for the TF Demo App](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/examples/android#bazel):
 
   1. [Install Bazel and Android Prerequisites](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/examples/android#install-bazel-and-android-prerequisites).
-     It's easiest with Android Studio.
+     It's easy with Android Studio.
 
-      - You'll need at least SDK version 23.
+      - You'll need atleast SDK version 23.
```

Comment:
The original version is correct. `At least` is two words indicating the presence of the conjunction 'at' that indicates the bare minimum availability of something.

Reply:
generally "atleast" is written as a single word. I am changing it to the original.